examples/feature_alignment_example/client.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from logging import INFO
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+from flwr.common.logger import log
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader, TensorDataset
+
+from examples.models.logistic_regression import LogisticRegression
+from fl4health.clients.tabular_data_client import TabularDataClient
+from fl4health.utils.metrics import Accuracy, Metric
+
+
+class Mimic3TabularDataClient(TabularDataClient):
+    def __init__(
+        self, data_path: Path, metrics: Sequence[Metric], device: torch.device, id_column: str, target_column: str
+    ) -> None:
+        super().__init__(data_path, metrics, device, id_column, target_column)
+
+    def get_data_loaders(self, config: Config) -> Tuple[DataLoader, DataLoader]:
+        batch_size = self.narrow_config_type(config, "batch_size", int)
+        indices = np.random.permutation(self.aligned_features.shape[0])
+        shuffled_data = self.aligned_features[indices]
+        shuffled_targets = self.aligned_targets[indices]
+        split_percentage = 0.9
+        split_point = int(shuffled_data.shape[0] * split_percentage)
+
+        train_data = shuffled_data[:split_point]
+        val_data = shuffled_data[split_point:]
+        train_targets = shuffled_targets[:split_point]
+        val_targets = shuffled_targets[split_point:]
+
+        tensor_train_data = torch.from_numpy(train_data.toarray())
+        tensor_train_targets = torch.from_numpy(train_targets)
+        tensor_val_data = torch.from_numpy(val_data.toarray())
+        tensor_val_targets = torch.from_numpy(val_targets)
+
+        tensor_train_data = tensor_train_data.float()
+        tensor_train_targets = torch.squeeze(tensor_train_targets.long(), dim=1)
+        tensor_val_data = tensor_val_data.float()
+        tensor_val_targets = torch.squeeze(tensor_val_targets.long(), dim=1)
+
+        train_loader = DataLoader(
+            TensorDataset(tensor_train_data, tensor_train_targets), batch_size=batch_size, shuffle=True
+        )
+
+        val_loader = DataLoader(
+            TensorDataset(tensor_val_data, tensor_val_targets), batch_size=batch_size, shuffle=True
+        )
+        return train_loader, val_loader
+
+    def get_model(self, config: Config) -> nn.Module:
+        return LogisticRegression(self.input_dimension, self.output_dimension)
```

Comment:
In my PR, I'm forcing our BasicClient `setup_client` to move the model to the proper device so we don't have to in the `get_model` function, but as it is setup now, it's a good idea to move the model to `self.device` here. Otherwise you'll run into issues if this example is run on GPU